src/js/utils/BetaOptInUtil.js

```diff
@@ -0,0 +1,85 @@
+import Util from "./Util";
+
+// SDK sets/reads this property within package config.
+const BETA_OPTIN_PROPERTY_NAME = "beta-optin";
+const DOT_NOTATION_SCHEMA_PATH = "properties.service.properties.beta-optin";
```

Comment:
I don't think the `DOT_NOTATION_` prefix is necessary, it's also probably ok to omit `BETA_OPTIN_` as both constants are part of the **BetaOptIn**Util. #style #naming